WordSearchTest.java

```diff
@@ -0,0 +1,54 @@
+package assignment3;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Test;
+
+public class WordSearchTest {
+  WordSearch wordSearch = new WordSearch();
+
+  @Test
+  public void findAllWords() {
+    char[][] grid = {{'A', 'A', 'R'}, {'T', 'C', 'D'}};
+    HashSet<String> words = new HashSet<>();
+    words.add("CAR");
+    words.add("CARD");
+    words.add("CART");
+    words.add("CAT");
+    Dictionary dictionary = new Dictionary(words);
+
+    Set<String> wordsFound = WordSearch.findAllWords(grid, dictionary);
+
+    assert (wordsFound.contains("CAR"));

```

Comment:
Typically you would use one of the methods from [`org.junit.Assert`](http://junit.sourceforge.net/javadoc/org/junit/Assert.html) class.

While `assert` keyword kind of works (i.e. fails), methods from `Assert` are extremely useful and can provide additional information about what exactly went wrong.